### DISCUSSION TITLE
Listen for "Hey Google" and automatically close after conversation is over

### DISCUSSION THREAD

**Discussion by paddyboo999** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#299] [Ideas]
It would be awesome if just saying "Hey google" would trigger the assistant, with an option to maybe even change the wake word(s), and for the app to close to the tray after a conversation/request is over. Thanks!! <3

**Comment by sumeshir26** [comment]
@paddyboo999  "Hey Google"/"Ok Google" support is already on the roadmap and will be implemented as soon as possible.

**Reply by paddyboo999** [reply]
What about the conversation ending automatically?

**Reply by sumeshir26** [reply]
@paddyboo999 If I get time, I will try working on it.

**Reply by paddyboo999** [reply]
Thanks!!

**Reply by giulianopires** [reply]
@sumeshir26 in fact i think it would be awesome if i could look like the old mac assistant. (https://github.com/vanshg/MacAssistant)
the icon is the same as the siri in the top bar and when you click it, the google assistant is already activated. then when it clicks again it closes.
i think it's a lot less work and is much better than having to click the keyboard shortcut and open an app window. besides, it should only be visible in the top bar, i think the app icon in the dock is unnecessary.

**Reply by im-coder-lg** [reply]
> @sumeshir26 in fact i think it would be awesome if i could look like the old mac assistant. (https://github.com/vanshg/MacAssistant)
> the icon is the same as the siri in the top bar and when you click it, the google assistant is already activated. then when it clicks again it closes.
> i think it's a lot less work and is much better than having to click the keyboard shortcut and open an app window. besides, it should only be visible in the top bar, i think the app icon in the dock is unnecessary.

I think you are suggesting something like a widget, right?

**Comment by im-coder-lg** [comment]
There might be one hitch. Your mic will be used by Google Assistant all the time and it might be a problem, especially to me. I might get the problems on the mic, like, I get a call on Skype or WhatsApp and the computer will realise that the mic is used by GAUDC and won't let you use it in the meantime. If it has a half-use permission or something, that would be really fine. I can't code it yet because I don't know how to code in JavaScript.
